lib/experimental/Utilities/PrivateBox/index.tsx

```diff
@@ -1,16 +1,30 @@
 import { usePrivacyMode } from "@/lib/privacyMode"
 import { cn } from "@/lib/utils"
+import { motion } from "framer-motion"
 import { FC, PropsWithChildren } from "react"
 
 export const PrivateBox: FC<PropsWithChildren> = ({ children }) => {
   const { enabled } = usePrivacyMode()
 
   return (
     <div
-      className={cn(enabled && "select-none blur-sm", "inline")}
+      className={cn(
+        "inline-flex ring-1 ring-inset ring-transparent transition-all duration-150",
+        enabled &&
+          "select-none overflow-hidden rounded-sm ring-f1-border-secondary"
+      )}
       aria-hidden={enabled}
     >
-      {children}
+      <motion.div
+        className="h-full w-full"
+        animate={{
+          opacity: enabled ? 0.3 : 1,
+          filter: enabled ? "blur(5px)" : "blur(0px)",
```

Comment:
We should increase this a little bit more, in the screenshot that I attached you could see the number

Reply:
Went with a different approach as we discussed, not using blur at all, which is prone to errors, and just hiding the text instead.